gene_expression_scatter_plot/support.html

```diff
@@ -30,10 +58,11 @@ <h3>3.Error code</h3>
                 '105': 'invalid json format in gene2 file'<br>
                 '106': 'invalid json format in information file'<br>
 				'108': 'invalid gene1, gene2, or information uploading order, please upload again by right order'<br></div>
-        <h3>4. How to contact us?</h3>
-        <div class="center-blue-higher-div"> &nbsp; &nbsp;You can send an email to <u>xumengqi&#64;zjnu.edu.cn</u> for help and we will reply to you as soon as possible.<br><i>Last modified:18 May 2019.</i></div>
-        <h3>5. Privacy Notice</h3>
+		<h3>5. How to contact us?</h3>
+        <div class="center-blue-higher-div"> &nbsp; &nbsp;You can send an email to <u>xumengqi&#64;zjnu.edu.cn</u> for help and we will reply to you as soon as possible.<p align="right" style="font-size: 12px; line-height: 0px;">Last modified:23 March 2020.</p></div>
+        <h3>6. Privacy Notice</h3>
         <div class="center-blue-higher-div"> &nbsp; &nbsp;Source files are only used for data processing of visualized scatter maps of gene expression. Programs do not store, disseminate or modify any source files.</div>
```

Comment:
@KerenHHH 

Change "or modify any source files" to "or modify any data files you uploaded".

Hui